server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -47,6 +49,24 @@ impl RemoteDataSynchroniser {
         Ok(())
     }
 
+    /// Request site info and persist it
+    pub(crate) async fn request_site_info(
+        &self,
+        connection: &StorageConnection,
+    ) -> Result<(), PostInitialisationError> {
+        info!("Requesting site info");
+        let site_info = self
+            .sync_api_v5
+            .get_site_info()
+            .await
+            .map_err(PostInitialisationError)?;
+
+        self.set_site_info(&connection, &site_info).unwrap();
```

Comment:
I suspect this `unwrap` is bad form, please help 😁

Reply:
yeah, unwrap would be bad here, it would kill the sync actor (sync thread). You would need to do something similar to RemotePullError for SetSiteInfoError, best if i show you, when you have time